### ISSUE TITLE
Default Content Text - Darker / More Contrast with Background

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#135]
The default text that appears most places has a gray colour, to meet accessibility standards it should have more contrast. Currently the gray is 3.54 to the white background. 

WCAG2 AA requires 4.5, and AAA requires 7

![color](https://user-images.githubusercontent.com/34379222/37601195-c8903520-2b5f-11e8-96f8-72e90b36b612.PNG)
![text](https://user-images.githubusercontent.com/34379222/37601196-c8a880da-2b5f-11e8-89e6-8fa0b76ba314.png)

**Comment by bmrgould** [comment]
![darker](https://user-images.githubusercontent.com/34379222/37729262-ea58d32e-2d12-11e8-8018-8faaed37bea6.png)
